protocol/consistencychecks.go

```diff
@@ -199,8 +195,8 @@ func (cc *ConsistencyChecks) verifyRegistration(msg *Response,
 func (cc *ConsistencyChecks) verifyKeyLookup(msg *Response,
 	uname string, key []byte) error {
 	df := msg.DirectoryResponse.(*DirectoryProof)
-	ap := df.AP
-	str := df.STR
+	ap := df.AP[0]
+	str := df.STR[0]
```

Comment:
I got it. I think even a malicious server has no reasons to such an invalid response. I will add a TODO here, since I fixed it in the next pulls.